app/models/taxon_parent_links_update.rb

```diff
@@ -0,0 +1,11 @@
+class TaxonParentLinksUpdate
```

Comment:
This is so we can re-use `PublishLinks`. This class has the same interface as `LinksUpdate`, which means `PublishLinks` doesn't have to be changed. Does that make sense?